packages/material-ui-system/src/index.d.ts

```diff
@@ -262,6 +263,424 @@ export const visuallyHidden: React.CSSProperties;
+  /**
+   * The **`border`** shorthand CSS property sets an element's border. It sets the values of `border-width`, `border-style`, and `border-color`.
+   *
+   * **Initial value**: `none`
+   *
+   * | Chrome | Firefox | Safari |  Edge  |  IE   |
+   * | :----: | :-----: | :----: | :----: | :---: |
+   * | **1**  |  **1**  | **1**  | **12** | **4** |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/border
+   */
+  border?: CSS.Property.Border | number;
+  /**
+   * The **`box-shadow`** CSS property adds shadow effects around an element's frame. You can set multiple effects separated by commas. A box shadow is described by X and Y offsets relative to the
+   * element, blur and spread radii, and color.
+   *
+   * **Initial value**: `none`
+   *
+   * | Chrome  | Firefox | Safari  |  Edge  |  IE   |
+   * | :-----: | :-----: | :-----: | :----: | :---: |
+   * | **10**  |  **4**  | **5.1** | **12** | **9** |
+   * | 1 _-x-_ |         | 3 _-x-_ |        |       |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/box-shadow
+   */
+  boxShadow?: CSS.Property.BoxShadow | number;
+  /**
+   * The **`font-weight`** CSS property specifies the weight (or boldness) of the font. The font weights available to you will depend on the `font-family` you are using. Some fonts are only
+   * available in `normal` and `bold`.
+   *
+   * **Initial value**: `normal`
+   *
+   * | Chrome | Firefox | Safari |  Edge  |  IE   |
+   * | :----: | :-----: | :----: | :----: | :---: |
+   * | **2**  |  **1**  | **1**  | **12** | **3** |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/font-weight
+   */
+  fontWeight?: CSS.Property.FontWeight | string;
+  /**
+   * The **`z-index`** CSS property sets the z-order of a positioned element and its descendants or flex items. Overlapping elements with a larger z-index cover those with a smaller one.
+   *
+   * **Initial value**: `auto`
+   *
+   * | Chrome | Firefox | Safari |  Edge  |  IE   |
+   * | :----: | :-----: | :----: | :----: | :---: |
+   * | **1**  |  **1**  | **1**  | **12** | **4** |
+   *
+   * @see https://developer.mozilla.org/docs/Web/CSS/z-index
+   */
+  zIndex?: CSS.Property.ZIndex | string;
+}
+
+/**
+ * Map of all available CSS properties (including aliases) and their raw value.
+ * Only used internally to map CCS properties to input types (responsive value,
+ * theme function or nested) in `SystemCssProperties`.
+ */
+export interface AllSystemCSSProperties
+  extends Omit<CSSProperties, keyof OverwriteCSSProperties>,
+    AliasesCSSProperties,
+    OverwriteCSSProperties {}
+
+export type SystemCssProperties<Theme extends object = {}> = {
+  [K in keyof AllSystemCSSProperties]:
+    | ResponsiveStyleValue<AllSystemCSSProperties[K]>
+    | ((theme: Theme) => ResponsiveStyleValue<AllSystemCSSProperties[K]>)
+    | SystemStyleObject;
+};
+
+/**
+ * The `SystemStyleObject` extends [style props](https://emotion.sh/docs/object-styles)
```

Comment:
This function doesn't depend on emotion, no? We should remove references to foreign documentation we don't control or depend on.

Reply:
Improved the description with the actual implementation. Let me know what you think.